spec/datadog/profiling/transport/http/adapters/net_integration_spec.rb

```diff
@@ -199,4 +199,26 @@
       end
     end
   end
+
+  def get_test_profiling_flush(code_provenance: nil)
```

Comment:
Replacement for the old `get_test_profiling_flush` that used to live in the `profiling/spec_helper.rb`. Will also be removed soon. (This is copy pasted in a couple of other places -- I thought it was easier this way because we only need to remember to delete everything from under `datadog/profiling/transport` and not touch any other files).